### PR TITLE
updated mpv related commandline option

### PR DESCRIPTION
**Pull request by CosmosAtlas**
Since the new mpv release (0.32.0) parsing for some command line options has been changed. 

More specifically, the option `force-media-title` is affected and needs to be changed to `--force-media-title=[media-title]` instead of the old format `--force-media-title [media-title]`

The updated version has been tested on Arch Linux.

### CODE REVIEW

**Comment by CosmosAtlas** [comment]
> Must change on all options.

Can you provide a test case which all options are used? Or multiple test cases so that all options will be covered?

In my personal usage, only the option `force-media-title` was involved. I want to make sure that the change of option format actually works.

**Comment by SeaHOH** [comment]
https://github.com/zhangn1985/ykdl/blob/bacfb0eeff9a8ad2859e902273f2e82fa09f2d31/ykdl/util/wrap.py#L33-L43

We also use other options to invoke MPV, you can run this command to test:
```
ykdl -p mpv https://v.qq.com/x/page/c3054acp6ef.html
```

**Comment by CosmosAtlas** [comment]
I've updated all options using the newer formatting requirements. And testing with the video link provides results in no errors.

File: ykdl/util/wrap.py [hunk 1]
```diff
@@ -34,13 +34,13 @@ def launch_player(player, urls, ext, **args):
         if ext == 'm3u8' and any(os.path.isfile(url) for url in urls):
             cmd += ['--demuxer-lavf-o', 'protocol_whitelist=[file,http,https,tls,rtp,tcp,udp,crypto,httpproxy]']
```
Comment (SeaHOH) [inline ykdl/util/wrap.py]: Also this one is.

Comment (CosmosAtlas) [reply ykdl/util/wrap.py]: I've updated this, but I don't have a source to test on it. Hopefully it works fine. : )

**Comment by SeaHOH** [comment]
Thanks very much!

**Pull request merged**